packages/blocks/person/package.json

```diff
@@ -76,8 +77,14 @@
     "image": "public/preview.svg",
     "examples": [
       {
+        "avatar": "https://i.pravatar.cc/300",
+        "employer": {
+          "name": "Bain & Co.",
+          "position": "General Manager of Insurance Claims"
+        },
+        "name": "Archibald Adams-Montgomery",
         "email": "alice@example.com",
-        "name": "Alice Alison"
+        "link": "https://alice.com/about"
```

Comment:
Perhaps this should be a more neutral URL such as `https://example.com/archibald` so we don't potentially send people to a website we're not interested in linking to

Reply:
Either I or @MaggieAppleton should be asked for a final UAT of this. That means someone set up to run HASH locally should sit down 1:1 or screenshare with one of us.